src/components/avatar-image.tsx

```diff
@@ -14,23 +15,33 @@ const StyledImg = styled.img`
 `;
 
 interface AvatarImageProps {
-  src?: string | null;
+  /**
+   * Can be Auth0 user (i.e. ‘me’) or data from Prisma.
+   *
+   * If picture is missing but email is present, gravatar URL is generated
+   */
+  user: {
+    name?: string | null | undefined;
+    picture?: string | null | undefined;
+    email?: string | null | undefined;
+  };
```

Comment:
In theory we can hoist all three params and then do `<AvatarImage {...user} size={42} />` instead of `<AvatarImage user={user} size={42} />`. I’m happy both ways.